packages/@ember/-internals/glimmer/index.ts

```diff
@@ -324,3 +324,4 @@ export {
   setComponentManager,
 } from './lib/utils/managers';
 export { isSerializationFirstNode } from './lib/utils/serialization-first-node-helpers';
+export { uniqueId } from './lib/helpers/unique-id';
```

Comment:
Here, you'll need to update to re-export the default:

```suggestion
export { default as uniqueId } from './lib/helpers/unique-id';
```